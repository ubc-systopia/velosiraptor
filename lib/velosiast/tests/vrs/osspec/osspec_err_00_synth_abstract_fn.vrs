
osspec MyOs () {
    abstract fn foo() -> bool
    synth fn bar() -> bool
}

