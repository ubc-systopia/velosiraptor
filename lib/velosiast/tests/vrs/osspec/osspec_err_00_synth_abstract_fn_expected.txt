error: OS specification does not support abstract methods
     --> tests/vrs/osspec/osspec_err_00_synth_abstract_fn.vrs:3:5
      |
    3 |      abstract fn foo() -> bool
      |      ^^^^^^^^ remove this method or make it non-abstract
error: OS specification does not support synth methods
     --> tests/vrs/osspec/osspec_err_00_synth_abstract_fn.vrs:4:5
      |
    4 |      synth fn bar() -> bool
      |      ^^^^^ remove this method or make it non-abstract
error: could not compile due to 2 previous errors; 0 warnings emitted
